aspnetcore/signalr/hubcontext.md

```diff
@@ -0,0 +1,62 @@
+---
+title: SignalR HubContext
+author: rachelappel
+description: Learn how to use the ASP.NET Core SignalR HubContext service for sending notifications to clients from outside a hub.
+manager: wpickett
+monikerRange: '>= aspnetcore-2.1'
+ms.author: rachelap
+ms.custom: mvc
+ms.date: 06/13/2018
+ms.prod: aspnet-core
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/HubContext
+---
+# Send messages from outside a hub
+
+By [Mikael Mengistu](https://github.com/mikaelm12)
+
+
+The SignalR hub is the core abstraction for sending messages to clients connected to the SignalR server. It's also possible to send messages from other places in your app using the `IHubContext<T>` service. This article explains how to access a SignalR `IHubContext<T>` to send notification to clients from outside a hub.
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/signalr/hubcontext/sample/) [(how to download)](xref:tutorials/index#how-to-download-a-sample)
+
+## Get an instance of `IHubContext<T>`
+
+In ASP.NET Core SignalR, you can access an instance of `IHubContext<T>` via dependency injection. You can inject an instance of `IHubContext<your_hub_type>` into a controller, middleware, or other DI service. Use the instance to send messages to clients.
+
+> [!NOTE]
+> This differs from ASP.NET SignalR which used `GlobalHost` to provide access to the `IHubContext`. There was no built-in dependency injection system in ASP.NET.
```

Comment:
Missed an `<T>` here in `IHubContext` (it was generic in ASP.NET SignalR as well, so it still applies).

Also, removing the "Because" (while probably appropriate from a style perspective) kinda made these sentences seem stilted.

How about:

This differs from ASP.NET SignalR which used `GlobalHost` to provide access to the `IHubContext`. ASP.NET Core has a dependency injection framework that removes the need for this global singleton.

Reply:
Sounds good